### PR TITLE
Translate mongondb extension

### PR DESCRIPTION
**Pull request by Fan2Shrek**

### CODE REVIEW

File: reference/mongodb/architecture.xml [hunk 1]
```diff
@@ -0,0 +1,1002 @@
...
+    peut avoir besoin d'émettre des commandes <literal>hello</literal> supplémentaires
+    pour mettre à jour sa vue des topologies.
+   </para>
+  </section>
+
+  <section>
+   <title>Persistance des sockets (versions PHP avant 1.2.0)</title>
+
+   <para>
+    Les versions de l'extension avant 1.2.0 utilisent l'API des stream PHP pour
```
Comment (devnexen) [inline reference/mongodb/architecture.xml:181]: I would translate stream to flux (mentioned in TRADUCTIONS)

Comment (Fan2Shrek) [reply reference/mongodb/architecture.xml:181]: Whoops, I should check the TRADUCTIONS file more often ^^'

I think I use stream in previous PR's

File: reference/mongodb/architecture.xml [hunk 2]
```diff
@@ -0,0 +1,1002 @@
...
+        'addresses.$' => 'MyProject\Address',
+        'addresses.$.city' => 'MyProject\City',
+    ],
+]
+]]>
+    </programlisting>
+   </para>
+
+   <para>
+    Chaqu'un de ces trois types de données, ainsi que les mappages spécifiques
```
Comment (devnexen) [inline reference/mongodb/architecture.xml:620]: I guess it s more `Chacun` ?

File: reference/mongodb/security.xml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+<!-- EN-Revision: 4e0bf229045ff87d5f84469280bf0a6b195bfb8b Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<book xml:id="mongodb.security" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Sécurité</title>
+
+ <article xml:id="mongodb.security.request_injection">
+  <title>Attaques par injection de requêtes</title>
+  <para>
+   Si vous passez des paramètres <literal>$_GET</literal> (ou <literal>$_POST</literal>)
+   à vos requêtes, assurez-vous de les caster en chaînes de caractères avant.
```
Comment (devnexen) [inline reference/mongodb/security.xml:11]: `convertir` instead of `caster`

**Review by devnexen — APPROVED** [review]

**Pull request merged**